yarn.lock

```diff
@@ -8565,10 +8565,10 @@ matchdep@^2.0.0:
     resolve "^1.4.0"
     stack-trace "0.0.10"
 
-material-icons@^0.3.1:
-  version "0.3.1"
-  resolved "https://registry.yarnpkg.com/material-icons/-/material-icons-0.3.1.tgz#425e06e2448632d6249d6e1ffe2993682c5b171e"
-  integrity sha512-5Hbj76A6xDPcDZEbM4oxTknhWuMwGWnAHVLLPCEq9eVlcHb0fn4koU9ZeyMy1wjARtDEPAHfd5ZdL2Re5hf0zQ==
+material-design-icons@^3.0.1:
```

Comment:
are we sure this is the one we want to use? It looks [here](https://www.npmjs.com/package/material-design-icons) like it hasn't been updated in four years or so

Reply:
I'm wondering if instead we should just copy files from the repo or add it as a submodule or something

Reply:
I thought we might want to use [this](https://www.npmjs.com/package/@material-ui/icons) one as it was published by Olivier Tassinari, the co-founder of Material UI.  It says the icons are packaged as React components though, so I'm not certain how we would use them in a Hugo template.

Reply:
Hmm... it looks like there is a 4.0.0 version which is 3 months old but nobody published it to npm, so we'll have to get the font outside of npm

Reply:
yeah it looks like they decided to stop publishing to npm at some point - this is the story with honestly every google-published library we've used like this :roll_eyes: idk why if bootstrap has been published and available the same ways for like 10 years stuff from google is so flaky and unreliable...

I think we could either copy down the files and check them in or we could add a git submodule for it if we want to stick with these material icons. @gumaerc I think those react ones are probably overkill for just including some icons in our hugo templates.

maybe we should look at switching away from material-icons entirely? we could try [font awesome](https://fontawesome.com/) or look at [bootstrap's icons](https://icons.getbootstrap.com/)

Reply:
As long as they fulfill our design requirements for @abdulkdawson , I think we should definitely consider other options.